src/main/java/frc/robot/commands/DepositCommand.java

```diff
@@ -0,0 +1,125 @@
+package frc.robot.commands;
+
+import com.kauailabs.navx.frc.AHRS;
+
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.commands.swerve.FollowPathCommand;
+import frc.robot.subsystems.drivetrain.BaseDrivetrain;
+import frc.robot.subsystems.drivetrain.BaseSwerveSubsystem;
+
+public class DepositCommand extends CommandBase {
+    private final BaseSwerveSubsystem driveSubsystem; // changed to BASE SWERVE SUBSYS to allow for getRobotPosition() 
+    private final AHRS ahrs; 
+
+    private final boolean red;
+
+
+    /*  all below setpoint constants are in inches
+        X axis --> long axis of field
+        Y axis --> short axis of field
+        origin at corner of blue grid 
+    */ 
+    private final double BLUE_X_SET = 86.0; 
+    private final double BLUE_Y_SET = 30.0;
+    private final double BLUE_R_SET = -90.0; // rotation in degrees
+    
+    private final double RED_X_SET = 564.0; 
+    private final double RED_Y_SET = 30.0;
+    private final double RED_R_SET = 90.0; // rotation in degrees
+
+    Pose2d blueSetPoint = new Pose2d(BLUE_X_SET, BLUE_Y_SET, Rotation2d.fromDegrees(BLUE_R_SET));
```

Comment:
These can eventually be `PlacePosition`s like what caroline is implementing on the `crautonomous` branch? They also shouldn't be hard-coded (pass them in as params instead).